gen/ioctl/list.c

```diff
@@ -95,7 +95,7 @@ struct sockaddr {
 #include <linux/joystick.h>
 #include <linux/kd.h>
 #include <linux/kcov.h>
-#if !defined(__arm__) && !defined(__powerpc64__) && !defined(__riscv) // various errors
+#if !defined(__arm__) && !defined(__loongarch64) && !defined(__powerpc64__) && !defined(__riscv) // various errors
```

Comment:
Isn't `__loongarch__` enough? Even if you want to restrict to 64-bit LoongArch you should probably use `__loongarch_lp64` as the `__loongarch64` symbol is deprecated.

Reply:
thanks for all your advice